EssentialsAntiBuild/src/com/earth2me/essentials/antibuild/EssentialsAntiBuildListener.java

```diff
@@ -163,6 +170,110 @@ public void onHangingBreak(final HangingBreakByEntityEvent event) {
         }
     }
 
+    @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+    public void onItemFrameInteract(final PlayerInteractEntityEvent event) {
+        final User user = ess.getUser(event.getPlayer());
+
+        if (!(event.getRightClicked() instanceof ItemFrame)) {
+            return;
+        }
+
+        if (prot.getSettingBool(AntiBuildConfig.disable_build) && !user.canBuild() && !user.isAuthorized("essentials.build") && !metaPermCheck(user, "place", Material.ITEM_FRAME)) {
```

Comment:
This should probably instead be using an 'interact' meta perm check.

Reply:
EDIT: This whole function looks like it is 'place' oriented but should instead be interact oriented.

Reply:
Joe is braindead: https://ptb.discordapp.com/channels/390942438061113344/395279435538104320/709899304831287387